src/SAWScript/Crucible/MIR/ResolveSetupValue.hs

```diff
@@ -135,6 +142,14 @@ typeOfSetupValue mcc env nameEnv val =
     MS.SetupTuple () vals -> do
       tys <- traverse (typeOfSetupValue mcc env nameEnv) vals
       pure $ Mir.TyTuple tys
+    MS.SetupSlice slice ->
+      case slice of
+        MirSetupSliceRaw{} ->
+          panic "typeOfSetupValue" ["MirSetupSliceRaw not yet implemented"]
```

Comment:
`MirSetupSliceRaw` is only used in `crucible-mir-comp` at the moment, so I didn't bother implemented support for it in SAW.